frontend/src/components/layout/navigation/whatsnew/WhatsNewMenu.tsx

```diff
@@ -310,7 +308,7 @@ export const WhatsNewMenu = (props: Props) => {
     },
   };
   // Only show its announcement if tracker removal is live:
-  if (flagIsActive(runtimeData.data, "tracker_removal")) {
+  if (flagIsActive(props.runtimeData, "tracker_removal")) {
```

Comment:
Oh, good one @lloan! It doesn't, but @maxxcrawford, you'll get a conflict here, which you'll want to resolve to this:

```suggestion
  if (flagIsActive(props.runtimeData, "tracker_removal")) {
```
```suggestion
  if (isFlagActive(props.runtimeData, "tracker_removal")) {
```